api/api.go

```diff
@@ -58,9 +58,9 @@ func CreateAndInitialise(bindAddr string, elasticSearchClient ElasticSearcher, e
 	httpServer.HandleOSSignals = false
 
 	go func() {
-		log.Debug("Starting search-query api...", nil)
+		log.Event(nil, "search-query api starting")
 		if err := httpServer.ListenAndServe(); err != nil {
-			log.ErrorC("search-query api http server returned error", err, nil)
+			log.Event(nil, "search-query api http server returned error", log.Error(err), log.ERROR)
```

Comment:
You don't need to explicitly pass in `log.ERROR` I believe the library work out the level for you (Im guessing based on an error opt being provided).

Reply:
Updated in all places where this was the case.